opm/autodiff/BlackoilModelBase.hpp

```diff
@@ -153,6 +153,8 @@ namespace Opm {
                           const WellModel&                well_model,
                           const NewtonIterationBlackoilInterface& linsolver,
                           std::shared_ptr< const EclipseState > eclState,
+                          std::shared_ptr< const Schedule> schedule,
```

Comment:
These two new arguments seem unnecessary to me, I cannot find anywhere they are used in the code. Seems to me a nice outcome of the refactoring!

Reply:
I will fix up this later today.